.eslintignore

```diff
@@ -9,3 +9,5 @@ jest.esm.setup.js
 
 # This directory contains input and output test fixtures, which contain generated code. It is not expected it to be well formatted.
 **/one-app-index-loader/*
+
+*.snap
```

Comment:
Why are we ignoring snaps now?

Reply:
I saw the big snap that likely triggered this decision. It's a whole generated file. I think it would be best to not use a snapshot for that. Just generate a baseline version of the file to compare it to.